<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MainWidget" name="mainWidget" native="true"/>
      <widget class="ViewerWidget" name="viewerWidget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Properties"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_ZoomFit"/>
    <addaction name="action_ZoomIn"/>
    <addaction name="action_ZoomOut"/>
    <addaction name="separator"/>
    <addaction name="action_Axes"/>
    <addaction name="action_Grid"/>
    <addaction name="action_Background"/>
   </widget>
   <widget class="QMenu" name="menuExamples">
    <property name="title">
     <string>Exa&amp;mples</string>
    </property>
    <addaction name="action_SimpleDrawing"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menuExamples"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar {
    background-color: rgb(80, 80, 80);
	color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QToolBar" name="fileBar">
   <property name="windowTitle">
    <string>File toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
  </widget>
  <widget class="QToolBar" name="viewBar">
   <property name="windowTitle">
    <string>View toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_ZoomFit"/>
   <addaction name="action_ZoomIn"/>
   <addaction name="action_ZoomOut"/>
   <addaction name="separator"/>
   <addaction name="action_Axes"/>
   <addaction name="action_Grid"/>
   <addaction name="action_Background"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu-qt/fileOpen.png</normaloff>:/icons/menu-qt/fileOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>About Visibility Analysis...</string>
   </property>
   <property name="toolTip">
    <string>About Visibility Analysis</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu-qt/fileNew.png</normaloff>:/icons/menu-qt/fileNew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu-qt/fileSave.png</normaloff>:/icons/menu-qt/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu-qt/fileSave.png</normaloff>:/icons/menu-qt/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save As</string>
   </property>
  </action>
  <action name="action_SimpleDrawing">
   <property name="text">
    <string>Simple Drawing</string>
   </property>
  </action>
  <action name="action_Axes">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/view_axes_24x24.png</normaloff>:/icons/menu/view_axes_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Axes</string>
   </property>
  </action>
  <action name="action_Grid">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/view_grid_24x24.png</normaloff>:/icons/menu/view_grid_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Grid</string>
   </property>
  </action>
  <action name="action_Background">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/view_background_24x24.png</normaloff>:/icons/menu/view_background_24x24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Background</string>
   </property>
  </action>
  <action name="action_ZoomFit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/view_zoom_fit_24x24.png</normaloff>:/icons/menu/view_zoom_fit_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit</string>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/view_zoom_in_24x24.png</normaloff>:/icons/menu/view_zoom_in_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/menu/view_zoom_out_24x24.png</normaloff>:/icons/menu/view_zoom_out_24x24.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="action_Properties">
   <property name="text">
    <string>Properties...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewerWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/ViewerWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/MainWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
